src/main/proto/wfa/measurement/internal/loadtest/labeled_event.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.loadtest;
+
+import "google/type/date.proto";
+
+option java_package = "org.wfanet.measurement.internal.loadtest";
+option java_multiple_files = true;
+
+// Input VID data for sketch generation within Correctness Test.
+message LabeledEvent {
+  // Identifier of the publisher.
+  int64 publisher_id = 1;
+
+  // Identifier of the event.
+  int64 event_id = 2;
+
+  // Type of sex.
+  enum Sex {
```

Comment:
{sex, age, social grad} is only used inside the sheet to get the query parameters. The data in the sheet only contains the integer demographics. In the query we will also use the integer values. So it seems that there is no need to parse the integers back to {sex, age, social grad}

Reply:
nvw, I see the updated sheet now.